Bottom: 304a650a3f34514901ea7fa59868e0ba7e817e57
Top:    da270f5d29f052c565b12f6da619ff49d7d36c68
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 16:14:27 +0000

Enable -Wextra (NOT yet -pedantic)


---

diff --git a/configure.ac b/configure.ac
index 058fa40..10d84b5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1143,7 +1143,7 @@ LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_L
 # if we have gcc then add -Wall
 if test "x$GCC" = "xyes"; then
 	# see about adding some extra checks if the compiler takes them
-	for flag in -Wall ; do
+	for flag in -Wall -Wextra; do #-pedantic
 		case " ${CFLAGS} " in
 			*\ ${flag}\ *)
 				# flag is already present
